src/sentry/api/endpoints/organization_issues_resolved_in_release.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.api.base import EnvironmentMixin
+from sentry.api.bases.organization import (
+    OrganizationEndpoint,
+    OrganizationPermission,
+)
+from sentry.api.helpers.releases import get_group_ids_resolved_in_release
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.group import GroupSerializer
+from sentry.models import Group
+
+
+class OrganizationIssuesResolvedInReleaseEndpoint(OrganizationEndpoint, EnvironmentMixin):
+    permission_classes = (OrganizationPermission, )
+
+    def get(self, request, organization, version):
+        """
+        List issues to be resolved in a particular release
+        ``````````````````````````````````````````````````
+
+        Retrieve a list of issues to be resolved in a given release.
+
+        :pparam string organization_slug: the slug of the organization the
+                                          release belongs to.
+        :pparam string version: the version identifier of the release.
+        :auth: required
+        """
+        group_ids = get_group_ids_resolved_in_release(organization, version)
+        groups = Group.objects.filter(
+            project_id__in=self.get_project_ids(request, organization),
+            id__in=group_ids,
+        )
+
+        context = serialize(
+            list(groups),
+            request.user,
+            GroupSerializer(
+                environment_func=self._get_environment_func(request, organization.id)
```

Comment:
is this going to need to support multiple environments?

Reply:
Hmm, I think it is. This seems relevant to a couple of places? I'll make a ticket for handling this separately

Reply:
https://getsentry.atlassian.net/browse/APP-956

Reply:
I added a `GroupSerializer` that supports multiple envs / uses snuba only: https://github.com/getsentry/sentry/blob/master/src/sentry/api/serializers/models/group.py#L550

but yeah, fine to do as a follow up if you prefer!

Reply:
Oh right, will look into using that as part of this. Thanks!